misc/theme-tukan/src/opnsense/www/themes/tukan/assets/stylesheets/main.scss

```diff
@@ -1,8078 +1,9818 @@
-@charset "UTF-8"
```

Comment:
I feel bad for asking, but this also doesn't look like SCSS or does it?

Reply:
@fichtner you are right:
```
sassc main.scss 
Error: Invalid CSS after "...rder: 1px solid": expected expression (e.g. 1px, bold), was "#565656cc;"
        on line 7960 of main.scss
>>   border: 1px solid #565656cc;
   --------------------^
```